libraries/incubator/ORM/Entity/EntityBuilder.php

```diff
@@ -385,11 +382,12 @@ public function handleStorage(Element $storage)
 					{
 						case 'csv':
 							$handler = '\Joomla\ORM\Storage\CsvProvider';
-							$param   = $parts[1];
+							break;
+						case 'orm':
+							$handler = '\Joomla\ORM\Storage\Doctrine\DoctrineProvider';
```

Comment:
There's no need for additional parameters.If you provide everything, that comes from outside, you have to check everything within the StorageProviders. Also, propagating a 'bunch' of parameters disguises, what the provider really needs - you have to read the source code of the provider to determine that. The `$param` approach lets you know from the signature of the provider's constructor.